requirements/requirements-python3.7.txt

```diff
@@ -38,6 +38,7 @@ alembic==1.4.2
 amqp==2.5.2
 analytics-python==1.2.9
 ansiwrap==0.8.4
+apache-beam==2.20.0
```

Comment:
This version is not supported yet. We support the Apache Beam 2.19 and earlier.

Reply:
That's another reason why we shoudl have it in setup.py as extra (with the appropriate limit). Updated it.